bindings/mqtt/0.2.0/message.json

```
@@ -1,16 +1,54 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "http://asyncapi.com/bindings/mqtt/0.2.0/message.json",
+  "$id": "http://asyncapi.com/bindings/mqtt/0.0.2/message.json",
   "title": "MQTT message bindings object",
   "description": "This object contains information about the message representation in MQTT.",
   "type": "object",
   "additionalProperties": false,
   "patternProperties": {
-    "^x-[\\w\\d\\.\\x2d_]+$": {
+    "^x-[\\w\\d\\.\\-\\_]+$": {
```

That's an invalid pattern. Should stay as it is.

```suggestion
    "^x-[\\w\\d\\.\\x2d_]+$": {
```